react-app/src/components/Banner/MMDLSpaBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import XIcon from "@/assets/X_dismiss_icon.svg";
+import InfoIcon from "@/assets/info_circle_icon.svg";
+
+const MMDLAlertBanner = () => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  const handleDismiss = () => {
+    setIsVisible(false);
+    localStorage.setItem("mmdlAlertDismissed", "true");
+  };
+
+  const goToFAQs = () => {
+    window.open("/faq#mmdl-section", "_blank");
+  };
```

Comment:
We should flesh out the local storage logic more. Why do we need to set local storage to `true` if the value `mmdlAlertDismissed` isn't even used anywhere? If I refresh the page, the banner isn't actually dismissed. 

Happy to huddle and discuss this a bit more

Reply:
I had it set to true because when I tested it with false, it did go away but once I dismissed it, it won't show anymore under the test login email. I can change it back to false.

Reply:
Just to clarify:
This banner is supposed to work with `localStorage`, correct?

Reply:
The function is being built out currently with Daniel's ticket. For this ticket purpose, the functionality doesn't exist in the backend yet. I needed a way to show it disappearing based on the user signed in so I used localStorage. It should be changed to match Daniels functionality when its added.

Reply:
I removed it. It was for local testing for me.